### PR TITLE
Add test 'Sort Stores'

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Add test 'Sort Stores' to the functional campaign
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | `TEST_PATH="functional/BO/13_shopParameters/05_contact/stores/04_sortStores" URL_FO=shopUrl/ npm run specific-test`

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/21040)

### CODE REVIEW

File: tests/UI/campaigns/functional/BO/13_shopParameters/05_contact/stores/04_sortStores.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+          testIdentifier: 'sortByCityAsc', sortBy: 'city', sortDirection: 'up',
+        },
+      },
+      {
+        args: {
+          testIdentifier: 'sortByCityDesc', sortBy: 'city', sortDirection: 'down',
+        },
+      },
+
+      /* Sort by post code not working, skipping it
```
Comment (SimonGrn) [inline tests/UI/campaigns/functional/BO/13_shopParameters/05_contact/stores/04_sortStores.js:115]: We need to keep track of this somewhere.

**Review by SimonGrn — DISMISSED** [review]

File: tests/UI/campaigns/functional/BO/13_shopParameters/05_contact/stores/04_sortStores.js (hunk 1 above)
Comment (boubkerbribri) [reply tests/UI/campaigns/functional/BO/13_shopParameters/05_contact/stores/04_sortStores.js:115]: I will mention this in the issue that i just created (after merging the PR). I don't know if it's enough. WDYT ?

**Review by PierreRambaud — DISMISSED** [review]

**Review by SimonGrn — APPROVED** [review]

File: tests/UI/campaigns/functional/BO/13_shopParameters/05_contact/stores/04_sortStores.js (hunk 1 above)
Comment (SimonGrn) [reply tests/UI/campaigns/functional/BO/13_shopParameters/05_contact/stores/04_sortStores.js:115]: I guess.

**Review by Progi1984 — APPROVED** [review]

**Comment by atomiix** [comment]
Thanks @boubkerbribri

**Pull request merged**